.circleci/config.yml

```diff
@@ -1142,6 +1153,9 @@ workflows:
       - test-modularize-instance:
           requires:
             - build-linux
+      - test-esm-integration:
```

Comment:
do we need both the modularize-instance mode and the esm integration mode?

Reply:
For the time being I think it would be good.   Maybe we can delete the former completely once this lands.